src/sentry/templates/sentry/organization-members.html

```diff
@@ -85,7 +85,7 @@
         {% for member, needs_sso, has_2fa in member_list %}
           <tr {% if member.role == request.GET.role %}class="member-role-highlight"{% endif %}>
             <td class="table-user-info">
-              {% avatar member %}
+              {% avatar member.user %}
```

Comment:
so `OrganizationMember`s don't have a user when they are just in the 'invited' state. so i think we're going to need to change the avatar helper instead of passing `member.user` here